src/fenicsxconcrete/boundary_conditions/bcs.py

```diff
@@ -1,11 +1,13 @@
 """Easy definition of Dirichlet and Neumann BCs."""
 
 import typing
-import numpy.typing as npt
 
 import dolfinx
-import ufl
 import numpy as np
+import ufl
+from dolfinx.fem.bcs import DirichletBCMetaClass
+from dolfinx.fem.function import Constant, Function, FunctionSpace
+from dolfinx.mesh import Mesh
```

Comment:
why not just `import dolfinx`?
(I am not against this, but having one import is easier to handle I would say)

Reply:
sure

Reply:
I just saw that Mesh is not even used in the file :)